Fixy

babyccino opened the pull request:
Fixes a bug where Vim bindings would flash in the mouse context menu and then be replaced by the default keybindings. Also fixes those bindings not being usable while the mouse context menu was open.

Release Notes:

- Fixed bug where Vim bindings were not available when mouse context menu was open

maxdeviant commented:
It seems like there are multiple different things being addressed in this PR.

Could we split them up so that they can be reviewed/merged independently?

babyccino commented:
> It seems like there are multiple different things being addressed in this PR.
> 
> Could we split them up so that they can be reviewed/merged independently?

Yep. I've removed the two styling changes

babyccino committed:
fix bug with keymaps flickering in mouse menu
